mirdata/datasets/acousticbrainz_genre.py

```diff
@@ -453,7 +438,7 @@ def download(self, partial_download=None, force_overwrite=False, cleanup=False):
                 )
                 if os.path.isdir(first_dir_path):
                     file_downloaded = True
-                    print(
+                    logging.info(
```

Comment:
Not covered :/ I guess this was like this before? Maybe ignore for now (I opened #440 to cover this stuff later)